### PR TITLE
Preferences tests

### PR DESCRIPTION
**Pull request by qadams**
Preferences test

### CODE REVIEW

File: tests/preferences/preferences.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/**
+ *
+ * Best practice: Add some info about the file at the top in a multi-line comment.
+ * This not only helps the next person using the file, but also gives you credit for writing it
+ * because you should also add your name as the author. Remove this after you have added your info.
+ *
+ * Default file headings can be configured in jet brains products so you don't have to write them out every time.
+ * I can show you how in person.
+ * - Jake
+ */
```
Comment (jakeharding) [inline tests/preferences/preferences.spec.ts]: Please read and remove the above comment^^^.

File: tests/preferences/PreferencesPageObject.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    super();
+    this.submitBtn = this.getElementByTid("submitBtn");
+    this.addAnotherBtn = this.getElementByTid("addAnotherBtn");
+    this.enterBeerInput = new Input("enterBeerInput");
+    this.enterAbvHigh = new Input("enterAbvHigh");
+    this.enterAbvLow = new Input("enterAbvLow");
+    this.enterIbuHigh = new Input("enterIbuHigh");
+    this.enterIbuLow = new Input("enterIbuLow");
+    this.enterTextArea = this.getElementByTid("enterTextArea").all(by.tagName("textarea"))
+    // TODO Instantiate the other objects needed for other elements on the page.
```
Comment (jakeharding) [inline tests/preferences/PreferencesPageObject.ts]: Please remove the TODOs if they are done.

File: tests/preferences/preferences.spec.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    page.enterAbvLow.enterText("Low")
+    page.enterIbuHigh.enterText("13")
+    page.enterIbuLow.enterText("14")
+    page.enterTextArea.sendKeys("I LIKE THE CITRUS IN MY BEER! Blue Moon is my most favorite out of any other beer. Testing the submit button")
+    page.submitBtn.click();
+    page.expectErrorIsDisplayed();
+  });
+
+  it('Test add another button', () => {
+    //TODO enter some input to the fields using the page object
```
Comment (jakeharding) [inline tests/preferences/preferences.spec.ts]: Remove TODOs if they are done please.

File: tests/preferences/preferences.spec.ts [hunk 4]
```diff
@@ -1,14 +1,3 @@
-/**
- *
- * Best practice: Add some info about the file at the top in a multi-line comment.
- * This not only helps the next person using the file, but also gives you credit for writing it
- * because you should also add your name as the author. Remove this after you have added your info.
- *
- * Default file headings can be configured in jet brains products so you don't have to write them out every time.
- * I can show you how in person.
- * - Jake
- */
```
Comment (jakeharding) [inline tests/preferences/preferences.spec.ts]: Did you read this before you removed it?

File: tests/preferences/PreferencesPageObject.ts [hunk 5]
```diff
@@ -1,12 +1,3 @@
-/**
- * PreferencesPageObject.ts
- *
- * Created by jake
- * Created on 2/18/18
- *
- * Preferences page object.
- */
```
Comment (jakeharding) [inline tests/preferences/PreferencesPageObject.ts]: Please don't remove file headings other's have made. This file heading is credit for authorship of the code. Removing it is like removing their credit for the work. You can add your name under as a modifier or contributor, but you should never remove these.

Comment (qadams) [reply tests/preferences/PreferencesPageObject.ts]: Oh. I thought you wanted every comment removed

Comment (jakeharding) [reply tests/preferences/PreferencesPageObject.ts]: I referred to one specific comment in `preferences.spec.ts` and still have a question pending there. Please address it; it is not a rhetorical question.

File: tests/preferences/preferences.spec.ts (hunk 4 above)
Comment (qadams) [reply tests/preferences/preferences.spec.ts]: Nope.

Comment (qadams) [reply tests/preferences/preferences.spec.ts]: Actually I did read it but you said read and remove...

Comment (jakeharding) [reply tests/preferences/preferences.spec.ts]: At least you are honest. Technically, you created this file and I only renamed it and adjusted it a little bit. The meaning of the file heading was to urge you to add a file heading and get credit for your work, but if you don't want to that is fine.

**Pull request merged**